packages/combobox/src/Combobox/Combobox.spec.tsx

```diff
@@ -541,6 +541,31 @@ describe('packages/combobox', () => {
         }
       });
 
+      test('Clicking an option fires onChange', () => {
+        const onChange = jest.fn();
+        const { openMenu } = renderCombobox(select, {
+          onChange,
+        });
+        const { optionElements } = openMenu();
+        expect(optionElements).not.toBeUndefined();
+        const option3 = (optionElements as HTMLCollectionOf<HTMLLIElement>)[2];
+        act(() => {
+          userEvent.click(option3);
+        });
+
+        if (select === 'multiple') {
+          expect(onChange).toHaveBeenCalledWith(
+            expect.arrayContaining(['carrot']),
```

Comment:
is this correct - an array containing an array with the string 'carrot'?
I'd expect `arrayContaining('carrot')`

Reply:
yeah it's confusing. Its essentially "array containing these elements: [...]"